dashboards/service-level.jsonnet

```diff
@@ -133,12 +126,17 @@ stdlib.dashboard(
       {
         query: 'slo_errors_per_request:ratio_rate1h{pipeline=~"$pipeline", service=~"$service", customer=~"$customer", installation=~"$installation", cluster_id=~"$cluster_id"}',
         legend: '{{service}} / {{installation}} / {{cluster_id}} (1h)',
+      },
+      {
+        query: 'min(slo_target_high{service=~"$service", installation=~"$installation", cluster_id=~"$cluster_id"}) by (service)',
```

Comment:
What's the `min` necessary for?

Having `pipeline` as a filter here would probably be good too